Automattic/wp-calypso
# Improve ResponsiveToolbarGroup: Fix responsive collapsing.

Related to https://github.com/Automattic/wp-calypso/issues/80492
Fixes https://github.com/Automattic/wp-calypso/issues/80555

## Proposed Changes

* Changed styles to make the toolbar use all the available width on the Design Picker
* Changed how the items are arranged to use all the available space.

|Before|After|
|----|----|
|![Captura de Pantalla 2023-08-17 a las 13 49 09](https://github.com/Automattic/wp-calypso/assets/1989914/0ba75775-5cd5-4185-b918-eaf69aedc5f9)|![Captura de Pantalla 2023-08-17 a las 13 48 42](https://github.com/Automattic/wp-calypso/assets/1989914/aabc0c52-1804-4c96-8f59-b6cf4710d37c)|

Before:

https://github.com/Automattic/wp-calypso/assets/1989914/53263466-88ab-4a2d-9cf0-5b8214e8db29

After:


https://github.com/Automattic/wp-calypso/assets/1989914/1f155886-74be-4c8d-9d84-5bcb8c68f5b5




## Testing Instructions



* Open the Calypso Livelink
* Go to the Design Picker
* You should see that the toolbar uses all the available screen width.
* You should see that it's responsive and works as expected.

## Pre-merge Checklist
